### PR TITLE
EXP-676 - Esconder opções de Aba quando cliente é Link.me

### PR DESCRIPTION
**Pull request by LucasRichter**
## Contexto
Quando a conta é do Link.me ele não tem acesso as Chaves de api e a Dados Bancarios. Com isso, não há necessidade de renderizar essas tabs. Esse PR esconde-lás nessa situação.

## Checklist
- [x] Esconder a tab de "Chaves de API" na tela de "Configurações"
- [x] Esoncder a tab de "Dados Bancários" na tela de "Dados Cadastrais"

### Layout:
Antes:
image
Depois:
depois_1
Antes:
image
Depois:
depois_2

## Como testar?
- Fazer login com uma company Link.me e verificar se as tabs não estão sendo mostradas
- Fazer login com uma company normal e verificar se as tabs estão sendo mostrdas.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5ba1242b66e3a65d3f3b887954deb7e798d2e227
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5ba1242b66e3a65d3f3b887954deb7e798d2e227
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5ba1242b66e3a65d3f3b887954deb7e798d2e227
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5ba1242b66e3a65d3f3b887954deb7e798d2e227
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/Settings/User/index.js [hunk 1]
```diff
@@ -56,18 +48,44 @@ const UserSettings = ({
...
+      setTabIndexByName({
+        accountInfo: 0,
+        password: 2,
+        teamInfo: 1,
+      })
+    }
+  }, [company, hideBankAccount])
 
+  if (hideBankAccount) {
+    tabs.splice(1, 1)
```
Comment (Hrfreire) [inline packages/pilot/src/containers/Settings/User/index.js:87]: Não seria melhor inserir a tab de banking info, caso seja necessário? ao invés de tirar. Pq da forma que tá, caso os indíces mudem e a gente esqueça de atualizar esse splice, pode acabar removendo uma aba errada.

acho que algo assim:

```suggestion
    tabs.splice(tabIndexByName.bankingInfo, 0, 'pages.settings.user.tabs.banking_info')
```

e lá da definição de tabs, tirar a linha do banking info do array.

File: packages/pilot/src/containers/Settings/User/index.js [hunk 2]
```diff
@@ -109,7 +129,7 @@ const UserSettings = ({
               bankAccountSelected={bankAccountSelected}
               bankAccountChangeActionDisabled={bankAccountChangeActionDisabled}
               bankAccountSelectedView={bankAccountSelectedView}
-              isPaymentLink={isPaymentLink(company)}
+              isPaymentLink={hideBankAccount}
```
Comment (Hrfreire) [inline packages/pilot/src/containers/Settings/User/index.js:132]: O componente `BankingInfoTab` usa a flag `isPaymentLink` só pra decidir se mostra ou não o conteúdo. Como agora esse comportamento tá sendo feito do lado de fora, fazendo com que a aba não possa ser mostrada, não seria o caso de remover essa prop que na prática não vai ser mais utilizada?

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1a34372aac1cc24e5dc374d3594dd5580e2adf38
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1a34372aac1cc24e5dc374d3594dd5580e2adf38
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1a34372aac1cc24e5dc374d3594dd5580e2adf38
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1a34372aac1cc24e5dc374d3594dd5580e2adf38
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/Settings/User/index.js [hunk 3]
```diff
@@ -55,18 +45,93 @@ const UserSettings = ({
   team,
 }) => {
   const [tabIndex, setTabIndex] = useState(0)
-  const history = useHistory()
-
-  useEffect(() => {
-    if (history.location.state) {
```
Comment (Hrfreire) [inline packages/pilot/src/containers/Settings/User/index.js:61]: @LucasRichter acho que esse useEffect aqui tem que existir ainda, e de alguma forma mapear as keys (provavelment as keys que existiam no tabIndexByName) pro index correto. Pelo que entendi, existe algum caso de uso que essa tab vem no history.location e que a pag ja seria carregada em uma tab especifica.

Comment (LucasRichter) [reply packages/pilot/src/containers/Settings/User/index.js:61]: Eu removi pq não tem nenhum lugar do código em que ele faz um history.push pra esse url com state. E como estamos usando o react-router-dom, esse state só pode ser adicionado por dentro do código, então esse useEffect nunca será chamado.

**Review by diegodsgarcia — COMMENTED** [review]
Já que estamos dando uma boa refatorada nesse Settings, aproveitei já e criei uns diffs que remove um monte de erro que tava dando no console por causa de prop-type :)

```diff
diff --git a/packages/pilot/src/containers/Settings/Company/PixInfoTab/index.js b/packages/pilot/src/containers/Settings/Company/PixInfoTab/index.js
index ee2f332f..1fac1a96 100644
--- a/packages/pilot/src/containers/Settings/Company/PixInfoTab/index.js
+++ b/packages/pilot/src/containers/Settings/Company/PixInfoTab/index.js
@@ -37,10 +37,12 @@ const PixInfoTab = ({
           }
           icon={<IconInfo height={16} width={16} />}
         >
-          {error
-            ? t('pages.settings.company.pix.error')
-            : alertInfoMessage
-          }
+          <>
+            {error
+              ? t('pages.settings.company.pix.error')
+              : alertInfoMessage
+            }
+          </>
         </Alert>
       </CardContent>
       {!isTestEnvironment && (
diff --git a/packages/pilot/src/containers/Settings/Company/index.js b/packages/
pilot/src/containers/Settings/Company/index.js
index cc865be9..6b62782e 100644
--- a/packages/pilot/src/containers/Settings/Company/index.js
+++ b/packages/pilot/src/containers/Settings/Company/index.js
@@ -132,7 +132,7 @@ const CompanySettings = ({
       selected={tabIndex}
       variant="just-text"
     >
-      { tabItems.map(({ key }) => <TabItem text={t(`pages.settings.company.tab.${key}`)} />) }
+      { tabItems.map(({ key }) => <TabItem key={key} text={t(`pages.settings.company.tab.${key}`)} />) }
     </TabBar>
 
     <div className={style.tabsContainer}>
@@ -175,12 +175,12 @@ CompanySettings.propTypes = {
     error: PropTypes.string,
     loading: PropTypes.bool,
     success: PropTypes.bool,
-  }).isRequired,
+  }),
  deleteUserStatus: PropTypes.shape({
     error: PropTypes.string,
     loading: PropTypes.bool,
     success: PropTypes.bool,
-  }).isRequired,
+  }),
   environment: PropTypes.oneOf([
     'live',
     'test',
@@ -223,6 +223,8 @@ CompanySettings.defaultProps = {
   boletoDaysToAddInExpirationDate: null,
   boletoInstructions: null,
   company: null,
+  createUserStatus: null,
+  deleteUserStatus: null,
   hasPixKey: false,
   pixError: false,
   pixLoading: false,
diff --git a/packages/pilot/src/pages/CompanySettings/index.js b/packages/pilot/src/pages/CompanySettings/index.js
index e0de822c..7b2baf38 100644
--- a/packages/pilot/src/pages/CompanySettings/index.js
+++ b/packages/pilot/src/pages/CompanySettings/index.js
@@ -663,7 +663,7 @@ CompanySettingsPage.propTypes = {
   }),
   defaultRecipient: PropTypes.shape({
     automatic_anticipation_1025_delay: PropTypes.number.isRequired,
-  }).isRequired,
+  }),
   fees: PropTypes.shape({
     anticipation: PropTypes.number,
     antifraud: PropTypes.number,
@@ -686,6 +686,7 @@ CompanySettingsPage.defaultProps = {
   acquirers: [],
   anticipationType: '',
   company: null,
+  defaultRecipient: null,
   fees: {},
   user: null,
 }

```

**Review by Hrfreire — APPROVED** [review]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: b02fda8d27641ae95b175ec2cc0161fa818c3060
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: b02fda8d27641ae95b175ec2cc0161fa818c3060
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: b02fda8d27641ae95b175ec2cc0161fa818c3060
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: b02fda8d27641ae95b175ec2cc0161fa818c3060
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: b02fda8d27641ae95b175ec2cc0161fa818c3060
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: b02fda8d27641ae95b175ec2cc0161fa818c3060
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: b02fda8d27641ae95b175ec2cc0161fa818c3060
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b31f40165b57400b338a5b14beb1c98747b35f9c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: b02fda8d27641ae95b175ec2cc0161fa818c3060
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbc162f547ca06502c384323cace04eb8cfc4fcc
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbc162f547ca06502c384323cace04eb8cfc4fcc
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbc162f547ca06502c384323cace04eb8cfc4fcc
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbc162f547ca06502c384323cace04eb8cfc4fcc
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fac2399b58dd556e7f98f11f66714877fd2aa521
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fac2399b58dd556e7f98f11f66714877fd2aa521
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fac2399b58dd556e7f98f11f66714877fd2aa521
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fac2399b58dd556e7f98f11f66714877fd2aa521
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Review by diegodsgarcia — APPROVED** [review]

**Review by Hrfreire — APPROVED** [review]

**Pull request merged**